packages/components/src/palette-edit/test/index.tsx

```diff
@@ -23,6 +24,8 @@ describe( 'getNameForPosition', () => {
 		const elements = [
 			{
 				slug: 'test-color-1',
+				color: '#ffffff',
+				name: 'Test Color 1',
```

Comment:
The [Color type](https://github.com/WordPress/gutenberg/pull/47764/files#diff-532dfa00a13a98be8d7f9bb3569fd9437ffa5e9988d7a7cc5031ff28d659cf46R11-R15) should have a `color` and `name` property.